src/main/java/core/basesyntax/dao/impl/UserDaoImpl.java

```diff
@@ -12,21 +15,63 @@ public UserDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public User create(User entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't create user entity " + entity, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public User get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.get(User.class, id);
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get user by id = " + id + " from DB", e);
+        }
     }
 
     @Override
     public List<User> getAll() {
-        return null;
+        try (Session session = factory.openSession()) {
+            Query<User> query = session.createQuery("FROM User", User.class);
+            return query.getResultList();
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get all users from DB", e);
+        }
     }
 
     @Override
     public void remove(User entity) {
-
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.delete(entity);
```

Comment:
let's be consistent in the code. use same methods throughout all common methods, in this case either `remove()` or `delete()`

Reply:
ok